plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -4,8 +4,8 @@
    
    <extension point="org.eclipse.wst.common.project.facet.core.facets">        
     <project-facet id="com.google.cloud.tools.eclipse.appengine.facet">
```

Comment:
You can put "conflicts" requirement on the flex facet that it conflicts with  the standard facet. I actually put it in and took it out for 2 reasons: (i) standard and flex depend on different versions of java (7 and 8 respectively), so that prevents users from adding both facets to a project, (ii) I was thinking about compat, I am not quite sure of how it works but theoretically, a user can deploy a standard project to a flex environment